tools/dashboard/README.md

```diff
@@ -1,5 +1,10 @@
 ![freeCodeCamp Social Banner](https://s3.amazonaws.com/freecodecamp/wide-social-banner.png)
 
-# contribute
+# Contribute
+
+Tools to help maintain [freeCodeCamp.org](https://www.freecodecamp.org)'s Open Source Codebase on GitHub. We are currently working on creating a tools dashboard for helping you review PRs and translations. Hangout with us in the [contributor's chat room](https://gitter.im/FreeCodeCamp/Contributors) to learn more. 
+
+
+
+
```

Comment:
```suggestion
```

Can you cleanup these extraneous lines. Thanks.

Reply:
Committed the new changes! Thank you.